packages/api-clients/graphql-client/src/api-client-utilities/validations.ts

```diff
@@ -25,6 +25,7 @@ export function validateDomainAndGetStoreUrl({
   } catch (_error) {
     throw new Error(
       `${client}: a valid store domain ("${storeDomain}") must be provided`,
+      {cause: _error},
```

Comment:
can we remove the `_` in `_error` because it's used now. `_` indicates that it's not used.